config.json

```
@@ -4,7 +4,7 @@
     "namespace": "STPipe\\ExternalModule",
     "framework-version": 15,
     "compatibility": {
-        "redcap-version-min": "14.0.2",
+        "redcap-version-max": "14.9.6",
```

After thinking hard about how to protect our module users from broken code, I think the answer is to allow people to use functional even if it might be broken on some systems. 

I think we should say 

```suggestion
        "redcap-version-max": "15.0.0",
```

On the next release, we should say, 
```
        "redcap-version-min": "14.5.35",
```

This combination will allow LTS users running 14.5.35 - 15.x to run broken code if they run this release. After we make the next release, it will allow STD users to run broken code if they are running 14.5.35 - 15.0.1

---

Right, there isn't a better combination of any max and min versions that'll work well on both LTS and STD versions.

---

The only closest one is 
```
 "redcap-version-max": "14.9.6",
 "redcap-version-min": "14.3.7"
```